docs/open-source/quickstart.mdx

```diff
@@ -195,50 +195,6 @@ m.delete_all(user_id="alice") # Delete all memories
 m.reset() # Reset all memories
 ```
 
-## Run Mem0 Locally
```

Comment:
Can you please just link to the above doc from this section and not remove this section? thanks

Reply:
It's done, please check.